src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,18 +23,50 @@ public class RobotContainer {
 	// Replace with CommandPS4Controller or CommandJoystick if needed
 	private final CommandXboxController driverController = new CommandXboxController(
 			OperatorConstants.DRIVER_CONTROLLER_PORT);
+	private final CommandXboxController XboxController = new CommandXboxController.getHID();
+	private final CommandXboxController kYButton = new XboxController.getYbutton();
+
 
 	// The robot's subsystems and commands are defined here...
 	private final Drivetrain drivetrain = new Drivetrain(driverController);
 
+
 	/**
 	 * The container for the robot. Contains subsystems, OI devices, and commands.
 	 */
 	public RobotContainer() {
 		// Configure the trigger bindings
 		configureBindings();
```

Comment:
Let's take advantage of this function, and configure a button to activate a command when pressed.
We can use a `CommandXboxController` object to create a `Trigger` -> `driverController.y()`
We can then configure that `Trigger` to call a Command in the `Drivetrain` subsystem. See: https://github.com/Rundle-Robotics/2023-Charged-Up/blob/6c0f9dd39d4c51a59897b680f9d681ebdee34850/src/main/java/frc/robot/RobotContainer.java#L54